### PR TITLE
Small improvements to docs API

### PR DESCRIPTION
**Pull request by ricardoV94**
## Description

Small changes, just to get a feel for it.

### CODE REVIEW

**Comment by ricardoV94** [comment]
No docs preview around here @OriolAbril :( ?

**Comment by OriolAbril** [comment]
There is docs preview but no job that adds the link in the PR description. You have to go fish for the link between all the checks

imatge

For reference, the action that adds the link is https://github.com/pymc-devs/pymc/blob/main/.github/workflows/rtd-link-preview.yml

**Comment by ricardoV94** [comment]
Thanks, added the workflow @OriolAbril

File: doc/library/graph/op.rst [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
 .. _libdoc_graph_op:
 
-==============================================================
-:mod:`graph` -- Objects and functions for computational graphs
-==============================================================
+===========================================
+:mod:`op` -- Objects that define operations
+===========================================
 
 .. automodule:: pytensor.graph.op
```
Comment (OriolAbril) [inline doc/library/graph/op.rst:7]: I think `:members:` is missing, this generates https://pytensor--817.org.readthedocs.build/en/817/library/graph/op.html. See https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#usage

File: doc/library/graph/replace.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.. _libdoc_graph_replace:
+
+=============================================
+:mod:`op` -- High level graph transformations
```
Comment (OriolAbril) [inline doc/library/graph/replace.rst:4]: The title should be replace

File: doc/library/graph/op.rst (hunk 1 above)
Comment (ricardoV94) [reply doc/library/graph/op.rst:7]: Oh yes, I deleted `members` by mistake, also above

**Pull request merged**